vendir.yml

```diff
@@ -6,7 +6,7 @@ directories:
       - path: cilium
         git:
           url: https://github.com/giantswarm/cilium
-          ref: "main"
+          ref: "v1.15"
```

Comment:
We agreed to sync from separate minor branches, because that's what upstream is doing with the releases.